cg/store/crud/create.py

```diff
@@ -415,3 +416,13 @@ def link_case_to_order(order_id: int, case_id: int):
         session = get_session()
         session.execute(insert_statement)
         session.commit()
+
+    def add_illumina_flow_cell(self, flow_cell: IlluminaFlowCell) -> IlluminaFlowCell:
+        """Add a new Illumina flow cell to the status database as a pending transaction."""
+        if not self.get_illumina_flow_cell_by_internal_id(flow_cell.internal_id):
+            session = get_session()
+            session.add(flow_cell)
+            LOG.debug(f"Flow cell added to status db: {flow_cell.id}.")
```

Comment:
Hehe, I think this can be info 😉 
```suggestion
            LOG.info(f"Flow cell added to status db: {flow_cell.id}.")
```